lib/record_store/record.rb

```diff
@@ -31,6 +31,10 @@ def unquote(value)
       def ensure_ends_with_dot(fqdn)
         fqdn.end_with?(".") ? fqdn : "#{fqdn}."
       end
+
+      def ensure_ends_without_dot(fqdn)
+        fqdn.end_with?(".") ? fqdn.sub(/\.$/, '') : fqdn
```

Comment:
```suggestion
        fqdn.sub(/\.$/, '')
```

Would be sufficient, we're already using the regex to remove the ending dot if there is a ending dot!

Reply:
Done